README.md

```diff
@@ -41,9 +41,22 @@ dnf install libtool clang-devel bison cmake ninja-build samba flex texinfo glib2
 
 # Basic usage
 
-If you want to start up a QEMU VM running CheriBSD run `cheribuild.py run-riscv64-purecap -d` (-d means build all dependencies).
-If you would like the VM to have all userspace binaries to be built as plain RISC-V binaries instead of CHERI pure-capability ones use `cheribuild.py run-riscv64-hybrid -d`.
-This will build the CHERI compiler, QEMU, CheriBSD, create a disk image and boot that in QEMU.
+The `cheribuild.py` program is normally run from within the
```

Comment:
Maybe we should just add something like
```
The commandlines given below assume that you have added cheribuild.py to your $PATH, if not you will have to change to the cheribuild directory and use `./cheribuild`?
```

Reply:
Does anyone add cheribuild to their path?  All the examples I see imply going in there and using ./ but I'm flexible on what we say.

Reply:
I do, pretty sure Alex does, I think Alfredo does too. IMO it's the best way, I have ~/bin/cheribuild (and other prefixed ones that automatically pick up separate configs) symlinked to ~/cheri/cheribuild/cheribuild.py so I can just `cheribuild foo` wherever whenever without having to think.

Reply:
I think the main reason the examples all use `./` on the website is because it removes a step and thus is a quicker onboarding experience (and gives us the "clone and run, that's it" marketing), but it's not the best experience long-term.

Reply:
I symlink ~/cheri/cheribuild/cheribuild.py to ~/bin/cheribuild

Reply:
I've also got a symlink to ~/bin (and a debug-cheribuild.py one that automatically selects debug-cheribuild.json)